Lib/json/tool.py

```diff
@@ -31,7 +31,8 @@ def main():
     parser.add_argument('--sort-keys', action='store_true', default=False,
                         help='sort the output of dictionaries alphabetically by key')
     parser.add_argument('--json-lines', action='store_true', default=False,
-                        help='parse input using the jsonlines format')
+                        help='parse input using the jsonlines format. '
+                        'Use with --no-indent or --compact to produce valid jsonlines output.')
```

Comment:
@hongweipeng (author of https://github.com/python/cpython/pull/10051): do you think this is a worthwhile and accurate addition to the documentation now that indentation options are supported?

Reply:
Looks good. Thanks.